shared/util.js

```diff
@@ -77,6 +77,21 @@ class Util {
     return details;
   }
 
+  /**
+   * Given the entity classification dataset and a URL, identify the entity.
+   * @param {string} url
+   * @param {LH.Result.Entities=} entities
+   * @return {LH.Result.LhrEntity|string}
+   */
+  static getEntityFromUrl(url, entities) {
```

Comment:
the subtlety here kinda goes over my head..

so we use the **new** PSL-powered getrootdomain in core/c/entity-classification
but out in report we use this method which leverages the **boring**-basic getrootdomain..

ya?

this does fix #15623 ? my brain is losing track of when the url data is set vs tweaked for display...

Reply:
ok cool thanks.


> Did you give it a try? 

nope but i knew you would've.